### ISSUE TITLE
rchk issues in 0.6.13

### ISSUE THREAD

**Issue by paciorek** [nimble-dev/nimble#847]
@perrydv see https://raw.githubusercontent.com/kalibera/cran-checks/master/rchk/results/nimble.out
for rchk issues occurring in 0.6.13.

We'll need to fix these in addition to the log(2) issue before Brian Ripley's Feb 8 deadline.

**Comment by paciorek** [comment]
```
Package nimble version 0.6.13
Package built using 76015/R 3.6.0; x86_64-pc-linux-gnu; 2019-01-26 05:51:36 UTC; unix   
Checked with rchk version c132302c5d39a55c436a06df73e539a1bace7486
More information at https://github.com/kalibera/cran-checks/blob/master/rchk/PROTECT.md

Function new_EIGEN_EIGENCLASS
  [UP] calling allocating function EIGEN_EIGENCLASS_castDerivedPtrPtrToPairOfPtrsSEXP with a fresh pointer (<unnamed var:   %3 = alloca %struct.SEXPREC*, align 8> <arg 1>) /unknown:0

Function new_EIGEN_SVDCLASS
  [UP] calling allocating function EIGEN_SVDCLASS_castDerivedPtrPtrToPairOfPtrsSEXP with a fresh pointer (<unnamed var:   %3 = alloca %struct.SEXPREC*, align 8> <arg 1>) /unknown:0

Function new_NIMBLE_ADCLASS
  [UP] calling allocating function NIMBLE_ADCLASS_castDerivedPtrPtrToPairOfPtrsSEXP with a fresh pointer (<unnamed var:   %3 = alloca %struct.SEXPREC*, align 8> <arg 1>) /unknown:0

Function new_OptimControlNimbleList
  [UP] calling allocating function OptimControlNimbleList_castDerivedPtrPtrToPairOfPtrsSEXP with a fresh pointer (<unnamed var:   %3 = alloca %struct.SEXPREC*, align 8> <arg 1>) /unknown:0

Function new_OptimResultNimbleList
  [UP] calling allocating function OptimResultNimbleList_castDerivedPtrPtrToPairOfPtrsSEXP with a fresh pointer (<unnamed var:   %3 = alloca %struct.SEXPREC*, align 8> <arg 1>) /unknown:0
```

**Comment by paciorek** [comment]
Fixed by PR #850
